platforms/platform-alexa/src/AlexaHandles.ts

```diff
@@ -85,4 +85,12 @@ export class AlexaHandles {
       platforms: ['alexa'],
     };
   }
+
+  static onCanFulfillIntent(): HandleOptions {
```

Comment:
Could we rename this to `onCanFulfillIntentRequest`?

Reply:
Done!

Reply:
Thanks!